qemu/tests/cfg/hv_info_check.cfg

```diff
@@ -0,0 +1,6 @@
+- hv_info_check:
+    type = hv_info_check
+    required_qemu = [6.1.0,)
+    cpu_model_flags += 'hv-reset,hv-crash'
```

Comment:
`start_vm = no`?

Reply:
no， we should start a guest.

Reply:
@menli820 I am asking because your test case description is like "run following cmd on host: # /usr/libexec/qemu-kvm -machine q35,accel=kvm -qmp tcp:0:4444,server,nowait", and you don't really have interact with your guest. But you are right we at least need a qemu instance so we do need `start_vm = yes`.